src/jit/codegencommon.cpp

```diff
@@ -11276,6 +11276,25 @@ bool Compiler::IsRegisterPassable(GenTreePtr tree)
 {
     return IsRegisterPassable(gtGetStructHandleIfPresent(tree));
 }
+
+// Returns true if the type is returned in multiple registers
```

Comment:
CC: Please use the function header from the Jit  Coding Convention document. It should start with:
//------------------------------------------------------------------------
// <Function name>: <Short description of the function>
//